src/core/components/auth/authorization-popup-perAPI.jsx

```diff
@@ -0,0 +1,47 @@
+import React from "react"
+import PropTypes from "prop-types"
+
+export default class AuthorizationPopupPerAPI extends React.Component {
```

Comment:
This isn't really a popup is it? Given it is displayed inline and explains the API accessibility, how about we name this component simply as `APIAccessDetails`?

Reply:
Yaa sure, I will rename this component as APIAccessDetails

Reply:
Resolved.